tests/spec/core/inlines-spec.js

```diff
@@ -367,4 +367,31 @@ describe("Core - Inlines", () => {
       "https://infra.spec.whatwg.org/#map-iterate"
     );
   });
+
+  it("processes {{ forContext/term }} IDL", async () => {
+    const body = `
+      <section>
+        <p id="link1">{{ Window.event }}</p>
+        <p id="link2">{{ Window/event }}</p>
```

Comment:
Wondering if we want to support  this now
```
{{
  Window
  /
  event
}}
```

Reply:
Yes, because HTML Tidy might produce some variation of that.